### PR TITLE
Simplify "Getting Started" tutorial

### PR DESCRIPTION
**Pull request by juliusv**
This gets rid of a few config options and flags that just clutter the
tutorial for beginners and add little value when getting started.

### CODE REVIEW

File: content/docs/introduction/getting_started.md [hunk 1]
```diff
@@ -34,23 +34,13 @@ useful in practice, it is a good starting example. Save the following basic
...
-    monitor: 'codelab-monitor'
-
 # A scrape configuration containing exactly one endpoint to scrape:
 # Here it's Prometheus itself.
 scrape_configs:
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'prometheus'
-
-    # Override the global default and scrape targets from this job every 5 seconds.
+    # Scrape targets from this job every 5 seconds.
```
Comment (brian-brazil) [inline content/docs/introduction/getting_started.md]: I'd keep the global and drop this

Comment (RichiH) [inline content/docs/introduction/getting_started.md]: Agreed.

File: content/docs/introduction/getting_started.md [hunk 2]
```diff
@@ -34,23 +34,13 @@ useful in practice, it is a good starting example. Save the following basic
...
-  external_labels:
-    monitor: 'codelab-monitor'
-
 # A scrape configuration containing exactly one endpoint to scrape:
 # Here it's Prometheus itself.
 scrape_configs:
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'prometheus'
-
-    # Override the global default and scrape targets from this job every 5 seconds.
```
Comment (RichiH) [inline content/docs/introduction/getting_started.md]: Keep this, it gives relevant context.

File: content/docs/introduction/getting_started.md [hunk 3]
```diff
@@ -65,26 +55,19 @@ To start Prometheus with your newly created configuration file, change to your
...
 ```language-bash
-# Start Prometheus.
-# By default, Prometheus stores its database in ./data (flag -storage.local.path).
-./prometheus -config.file=prometheus.yml
+./prometheus
 ```
 
+By default, Prometheus stores its database in `./data` (`-storage.local.path` flag)
+and loads its configuration from `prometheus.yml` (`-config.file` flag).
+
```
Comment (RichiH) [inline content/docs/introduction/getting_started.md]: I would mention config first and data second.

File: content/docs/introduction/getting_started.md [hunk 4]
```diff
@@ -186,16 +169,13 @@ section in your `prometheus.yml` and restart your Prometheus instance:
 
 ```
 scrape_configs:
-  - job_name:       'example-random'
-
-    # Override the global default and scrape targets from this job every 5 seconds.
+  - job_name: 'example-random'
+    # Scrape targets from this job every 5 seconds.
     scrape_interval: 5s
```
Comment (RichiH) [inline content/docs/introduction/getting_started.md]: Maybe kill the 5s so people see that the time interval for their data is different.

File: content/docs/introduction/getting_started.md [hunk 5]
```diff
@@ -186,16 +169,13 @@ section in your `prometheus.yml` and restart your Prometheus instance:
...
-
     static_configs:
       - targets: ['localhost:8080', 'localhost:8081']
         labels:
           group: 'production'
```
Comment (RichiH) [inline content/docs/introduction/getting_started.md]: Adding a short comment that this will add group=production and canary as a label could be useful.

**Comment by brian-brazil** [comment]
This was recently re-done.

**Pull request closed without merging**